packages/fastify/src/utils.ts

```diff
@@ -1,3 +1,28 @@
+import type { RequestAdapter } from '@clerk/backend';
+import { constants } from '@clerk/backend';
+import { parse } from 'cookie';
+import type { FastifyRequest } from 'fastify';
+
 export const getSingleValueFromArrayHeader = (value?: string[] | string): string | undefined => {
   return (Array.isArray(value) ? value[0] : value) || undefined;
 };
+
+export class FastifyRequestAdapter implements RequestAdapter {
```

Comment:
Do we require using a class here? 
If not, let's use a simple function instead. It should return a `RequestAdapter` eg:
```ts
export const fastifyRequestAdapter = (req): RequestAdapter => {
   ...
} 
```

Reply:
@nikosdouvlis No we don't require a class. It could be changed with a simple function.

Reply:
Let's use with simple functions then!